CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 2021-07-23
```

Comment:
tiny, non blocking: should we have a placeholder instead of the date here? It's obviously not the 23rd of July anymore and the date shall be stale unless adjusted before the first actual release

Reply:
@rhymes This date effectively is a placeholder. 🙂 When we cut our first real release, it will need to change whether it currently contains something that looks like a date or `[INSERT FIRST RELEASE DATE HERE]`. 😄 

I think part of the ongoing release process will need to be to add the next expected release date to this file in a PR, along with following up on PRs that didn't make it into a given release to have them move their changes to the latest unreleased version.